middle_end/flambda2/to_cmm/to_cmm_static.ml

```diff
@@ -327,6 +327,11 @@ let static_const0 env r ~updates ~params_and_body
       static_float_array_updates (C.symbol name) env updates 0 fields
     in
     env, R.update_data r float_array, e
+  | Block_like s, Empty_array ->
```

Comment:
Maybe we should add a comment here (and/or in other places) to recall that even float arrays have tag `0` when they are empty.

Reply:
There are I think at least two comments already but I have added another, since this is easy to forget.